concepticondata/conceptlists/Vejdemo-2016-167.tsv

```diff
@@ -0,0 +1,168 @@
+ID	CONCEPTICON_GLOSS	CONCEPTICON_ID	NUMBER	ENGLISH	Rate	Synonyms_average	SynonymsEnglish	SynonymsDanish	SynonymsDutch	SynonymsGerman	SynonymsSwedish	Frequency	Imageability	BrysbaertEtAl_concreteness	MutualInformation	Arousal	Senses	AgeOfAcquisition	WordClass
+Vejdemo-2016-167-1	ANIMAL	619	1	animal	3.4	0.469354827	0.241806402	1.014178482	0.325650558	0.453580902	0.311557789	97		4.61	6.4405	4.3	1	2.89	Noun
+Vejdemo-2016-167-2	ASH	646	2	ashes	2.65	0.275600166	0.214939024	0.12677231	0.054275093	0.151193634	0.830820771	9		4.92	2.463889			7.06362	Noun
+Vejdemo-2016-167-3	BACK	1291	3	back	4.29	0.543014562	0.429878049	1.140950792	0.325650558	0.403183024	0.415410385	148	5	4.33	5.2035	2.59	9	5.305215	Noun
+Vejdemo-2016-167-4	BAD	1292	4	bad	6.87	2.460135628	0.403010671	1.711426188	6.024535316	1.461538462	2.700167504	218	2.4	1.68	5848	4.86	14	2.79	Adjective
+Vejdemo-2016-167-5	BARK	1204	5	bark of a tree	3.79	0.275929132	0.161204268	0.063386155	0.434200743	0.201591512	0.519262982	8	5	4.52	2366	3.4	1		Noun
+Vejdemo-2016-167-6	BELLY	1251	6	belly	4.39	0.618613538	0.403010671	0.253544621	0.542750929	0.907161804	0.986599665	32		4.8	1.9295	3.75	5	4.05	Noun
+Vejdemo-2016-167-7	BIG	1202	7	big	3.41	2.079123691	2.06878811	4.500417014	1.356877323	0.75596817	1.713567839	1119	4.5	3.66	5388	4.33	13	2.89	Adjective
+Vejdemo-2016-167-8	BIRD	937	8	bird	2.15	0.237426512	0.161204268	0.443703086	0.325650558	0.100795756	0.155778894	63	6.4	5	7.5535	3.83	5	3.52	Noun
+Vejdemo-2016-167-9	BLACK	163	9	black	1.91							203	6.4	3.76	7.2595	3.58	5	3.56	Adjective
+Vejdemo-2016-167-10	BLOOD	946	10	blood	2.19	0.499507569	0.806021341	0.063386155		0.453580902	0.675041876	132	6.4	4.86	8647	5.76	5	4.89	Noun
+Vejdemo-2016-167-11	BONE	1394	11	bone	1.34	0.582561615		1.077564637	0.325650558	0.25198939	0.675041876	46	6.4	4.9	7.55	4.75	3	5.53	Noun
+Vejdemo-2016-167-12	CHILD (YOUNG HUMAN)	1304	12	child young	6.31	1.438809674	0.725419207	2.408673895	1.953903346	0.75596817	1.350083752	626	6.4	4.78	5838	5.33	4	5.15	Noun
```

Comment:
I remember that the original list had `_` and you removed most of those which is good. However, just having no delimiter at all looks weird, i.e. I'd prefer `bark (of a tree)` or `stone, rock` etc.

Reply:
You mean the delimiter of the English gloss? I guess that should reflect the original as faithfully as possible, right? So I'd say you better render this literally. Or did I not understand something here? So if the original elicitation gloss has a `_`, please keep it.

Reply:
Yes, specifically because in some cases, the underscore is still there. So please let's go for underscores here, as in the original.

Reply:
Sorry, for some reason I didn't think this through and actually recommended removing those back then. Apologies!